### DISCUSSION TITLE
What is the best way to keep my win 10, 64 bit, Thorium up to dat.

### DISCUSSION THREAD

**Discussion by BigDog044** [Alex313031/Thorium-WOA#3] [General]
I have Version 121.0.6167.204 (Official Build) (64-bit) installed in my C:\Program Files\Thorium directory. 
I want to keep up to date.  
Can or should I use the cmd line. Will Thorium automatically update. 
I saw the update for: The repo can be used on any distribution that supports .deb packaging. I have a windows terminal and tried using sudo rm -fv /etc/apt/sources.list.d/thorium.list && \
sudo wget --no-hsts -P /etc/apt/sources.list.d/ \
http://dl.thorium.rocks/debian/dists/stable/thorium.list && \
sudo apt update

sudo apt install thorium-browser 
This would not work.  I tried. 
I apologize if this is a stupid question?
Thank you for your time!  Like is short, we only have so much time! I am 60 years old. I am closer to the end than the beginning.

**Comment by gz83** [comment]
We apologize, but Thorium does not yet offer an automatic update feature for the Windows version.
